models/Page.js

```diff
@@ -8,6 +8,8 @@ const pageSchema = new mongoose.Schema({
   content: String,
   author: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
   isPublished: Boolean,
```

Comment:
Let's add a default `false` value here so that if we forgot to validate if it's exist it will fallback to `false`.